### DISCUSSION TITLE
v3.90

### DISCUSSION THREAD

**Discussion by MCC321-QC** [MCC321-QC/eMixedNiteMC#50] [Announcements]
- Fixed details panel when option for 1 column is enabled ([Asked ](https://github.com/MCC321-QC/eMixedNiteMC/issues/49)by [xCeeTee](https://github.com/xCeeTee))
    - All the item will now be visible
    - Margins still need to be worked on for when this option is enabled
- Added support for plugin [Theme Options](https://github.com/ashpynov/ThemeOptions?tab=readme-ov-file) from [ashpynov](https://github.com/ashpynov)
    - Added a preset to enable/disable rounded corner for Windows 11 ([Asked](https://github.com/MCC321-QC/eMixedNiteMC/issues/48) by [lionnesquaer](https://github.com/lionnesquaer))
    - Added a preset to change the Playnite logo of the theme
        - There is 5 Playnite logo you can choose from
        - eMixedNiteMC logo is enabled by default 
        - To re-enable the default Playnite logo, go in the plugin settings of [Theme Options](https://github.com/ashpynov/ThemeOptions)
- Re-added both details view value horizontal alignement options
    - Removed Global Text banner vertical and horizontal alignment options from [ThemeModifier](https://github.com/ashpynov/ThemeOptions?tab=readme-ov-file) file
        - You can still change these options manually in the constants file
- Changed default value of constants for Top Panel, Side Panel and Side Icon Panel
    - Default value for these constants is now "transparent"

**Full Changelog**: https://github.com/MCC321-QC/eMixedNiteMC/compare/v3.85...v3.90

<hr /><em>This discussion was created from the release <a href='https://github.com/MCC321-QC/eMixedNiteMC/releases/tag/v3.90'>v3.90</a>.</em>

**Comment by pavvelb** [comment]
Thank you for support!

**Reply by MCC321-QC** [reply]
Its my pleasure!
